### DISCUSSION TITLE
Dedicated renovate-config-validator image

### DISCUSSION THREAD

**Discussion by nejch** [renovatebot/renovate#29804] [Suggest an Idea]
The current default image is over 1GB uncompressed I think. For people using the [validator template](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/renovate-config-validator.gitlab-ci.yml?ref_type=heads#L4) in GitLab, I think this is a bit overkill as most of the time is spent just pulling the image (especially on distributed runners where things might not always be cached on that particular run).

Would you be open to having a separate `validator.Dockerfile` to build a tiny-ish image and then reference that at least in the GitLab CI template instead?

I'm pretty sure this really just needs the renovate NPM package installed inside a slim node image (but I might be missing something).

**Comment by MindTooth** [comment]
Could not help myself.  I see that Renovate is quite large still, but would save some MBs for sure.

```dockerfile
FROM cgr.dev/chainguard/wolfi-base

RUN apk add --no-cache renovate
```

```
renovate                                   latest 53273ad980e9  2 minutes ago     355MB
renovate/renovate                          latest 9929320f6b2e    5 hours ago    1.02GB
cgr.dev/chainguard/wolfi-base              latest 470833406092   15 hours ago    13.4MB
```

**Comment by jamietanna** [comment]
Raised https://github.com/renovatebot/renovate/issues/44237
